module/reset/include/mod_reset.h

```diff
@@ -0,0 +1,111 @@
+/*
+ * Arm SCP/MCP Software
+ * Copyright (c) 2015-2019, Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Description:
+ *     Power domain management support.
+ */
+
+#ifndef MOD_RESET_DOMAIN_H
+#define MOD_RESET_DOMAIN_H
```

Comment:
I would rename all the files to reset_domain. Reset is a bit ambiguous.

Reply:
Ok